server/src/main/java/org/apache/gravitino/server/web/rest/FilesetOperations.java

```diff
@@ -308,6 +345,15 @@ public Response dropFileset(
   @Produces("application/vnd.gravitino.v1+json")
   @Timed(name = "get-file-location." + MetricNames.HTTP_PROCESS_DURATION, absolute = true)
   @ResponseMetered(name = "get-file-location", absolute = true)
+  @AuthorizationExpression(
+      expression =
+          "METALAKE::READ_FILESET || CATALOG::READ_FILESET "
```

Comment:
Please use any expression like ANY(SELECT_TABLE,METALAKE,CATALOG,SCHEMA,TABLE)

Please refer to https://github.com/apache/gravitino/pull/7577/files